src/components/StatsTab/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Text, View } from "react-native";
+
+import getStatBarLengthAndColor from "../../_utils/getStatBarLengthAndColor";
+import type { Pokemon } from "../../types/interfaces/Pokemon";
+
+interface StatsTabProps {
+    viewedPokemon: Pokemon;
+}
+
+export default function StatsTab({ viewedPokemon }: StatsTabProps) {
+    return (
+        <View>
+            <Text className="pb-3 text-2xl font-chakra-bold">
+                Pokemon Stats
+            </Text>
+            <View className="flex flex-row w-full gap-2 ">
+                <View className="flex mr-2">
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        HP
+                    </Text>
+
+                    <Text className="mb-2 text-base text-gray-500 font-chakra ">
+                        Attack
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Defense
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Sp. Attack
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Sp. Defense
+                    </Text>
+                    <Text className="mb-2 text-base text-gray-500 font-chakra">
+                        Speed
+                    </Text>
```

Comment:
Same comment as above (check if we can consolidate common class names under one class name)